specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/resources.json

```diff
@@ -4714,6 +4823,19 @@
           },
           "description": "The API version."
         },
+        "defaultApiVersion": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The default API version."
+        },
+        "apiProfiles": {
```

Comment:
I don't see this is being referenced somewhere?

Reply:
what do you mean? it's just a regular property on the response payload, who should be referencing it? see example file

Reply:
Saw it, thanks.